docs/README.md

```diff
@@ -13,7 +13,6 @@ nebula.gl aspires to be an ultra-performant, fully 3D-enabled GeoJSON editing sy
 * Seamless integration with deck.gl and all geospatial deck.gl layers, allowing for GeoJSON editing to be interleaved with rich 3D visualizations.
 * Handle all aspects of event handling, including touch screen support.
 
-
```

Comment:
Didn't you reorganize these in three groups? Thought we were going to refresh these based on the draft post....